.github/workflows/golangci-lint.yml

```diff
@@ -20,8 +20,11 @@ jobs:
       - name: install Go
         uses: actions/setup-go@v2
         with:
-          go-version: 1.17.x
+          go-version: 1.18.x
```

Comment:
If I'm not mistaken, this file synced from Prometheus's main repo. 

We might need to change it in there, no?

https://github.com/prometheus/prometheus/blob/main/scripts/sync_repo_files.sh

Reply:
This is actually doing the sync from upstream, since we need some of the other updates in this PR to make it pass CI.

Reply:
https://github.com/prometheus/common/pull/379 was the last prombot sync, couldn't pass CI.